ratis-grpc/src/main/java/org/apache/ratis/grpc/server/GrpcLogAppender.java

```diff
@@ -86,7 +86,12 @@ private GrpcServerProtocolClient getClient() throws IOException {
   }
 
   private synchronized void resetClient(AppendEntriesRequest request, boolean onError) {
-    rpcService.getProxies().resetProxy(getFollowerId());
+    try {
+      rpcService.getProxies().getProxy(getFollowerId()).resetConnectBackoff();
```

Comment:
Use getClient() for rpcService.getProxies().getProxy(getFollowerId()).

Reply:
Thanks! It's duplicated.